docs/setup/install.md

```diff
@@ -0,0 +1,27 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster with version 1.21 or newer is running. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
+- The SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled. (In Kubernetes 1.22 or newer, the feature gate is enabled by default.)
+- The kubectl command-line tool has communication with your cluster.
+
+## Install Kueue
+
+To deploy Kueue in your cluster, run the following command:
```

Comment:
Actually, we should clarify that you need to clone the repo first.

Once we have a release, we'll truly have a one-line installation.